src/backend/CommandBuffer.cpp

```diff
@@ -69,10 +69,23 @@ namespace backend {
             return true;
         }
 
-        bool ComputeTextureCopyBufferSize(CommandBufferBuilder*, const TextureCopyLocation& location, uint32_t* bufferSize) {
+        bool ComputeTextureCopyBufferSize(CommandBufferBuilder*, const TextureCopyLocation& location, uint32_t rowPitch, uint32_t* bufferSize) {
             // TODO(cwallez@chromium.org): check for overflows
-            uint32_t pixelSize = static_cast<uint32_t>(TextureFormatPixelSize(location.texture->GetFormat()));
-            *bufferSize = location.width * location.height * location.depth * pixelSize;
+            *bufferSize = rowPitch * location.height * location.depth;
```

Comment:
If I remember correctly, I do think this is what the their helper function for computing the placed footprint does.